pkg/bff/db/models/v1/json.go

```diff
@@ -0,0 +1,30 @@
+package models
+
+import "google.golang.org/protobuf/encoding/protojson"
+
+// ProtocolBuffer JSON marshaling and unmarshaling ensures that the BFF JSON API works
+// as expected with protocol buffer models that are stored in the database.
+var (
+	pbencoder = protojson.MarshalOptions{
+		Multiline:       false,
+		Indent:          "",
+		AllowPartial:    true,
+		UseProtoNames:   true,
+		UseEnumNumbers:  false,
+		EmitUnpopulated: false,
+	}
+	pbdecoder = protojson.UnmarshalOptions{
+		AllowPartial:   true,
+		DiscardUnknown: true,
+	}
+)
+
+// MarshalJSON uses protojson with default marshaling options.
+func (r *RegistrationForm) MarshalJSON() ([]byte, error) {
+	return pbencoder.Marshal(r)
+}
+
+// UnmarshalJSON uses protojson with default unmarshaling options.
+func (r *RegistrationForm) UnmarshalJSON(data []byte) error {
+	return pbdecoder.Unmarshal(data, r)
+}
```

Comment:
Ensures the "rewiring" and "unwiring" of json to protocol buffers and back works as expected.